crates/hive-utils/src/madara/utils.rs

```diff
@@ -418,12 +417,12 @@ mod tests {
             // Set the storage record for the contract
             let contract_account_class_hash = env.kakarot().contract_account_class_hash;
             let genesis_address = StarknetContractAddress(Into::<StarkFelt>::into(genesis_address).try_into().unwrap());
-            let storage_record = StorageRecord {
-                nonce: Nonce(StarkFelt::from(0u8)),
-                class_hash: ClassHash(contract_account_class_hash.into()),
-                storage,
-            };
-            starknet.storage.insert(genesis_address, storage_record);
+
+            starknet.set_class_hash_at(genesis_address, ClassHash(contract_account_class_hash.into())).unwrap();
+            starknet.set_nonce(genesis_address, Nonce(StarkFelt::from(0u8)));
```

Comment:
i thought nonce were supposed to start at 1 for contract accounts

Reply:
we are actually setting the storage of an evm contract, I think the naming might be confusing.